packages/assets/imports.d.ts

```diff
@@ -1,232 +1,2 @@
-		'geo-diamond': string
-		'geo-ellipse': string
-		'geo-hexagon': string
-		'geo-octagon': string
-		'geo-oval': string
-		'geo-pentagon': string
-		'geo-rectangle': string
-		'geo-rhombus-2': string
-		'geo-rhombus': string
-		'geo-star': string
-		'geo-trapezoid': string
-		'geo-triangle': string
-		'geo-x-box': string
-		github: string
-		group: string
-		hidden: string
-		image: string
-		'info-circle': string
-		leading: string
-		link: string
-		'lock-small': string
-		lock: string
-		menu: string
-		minus: string
-		mixed: string
-		pack: string
-		page: string
-		plus: string
-		'question-mark-circle': string
-		'question-mark': string
-		redo: string
-		'reset-zoom': string
-		'rotate-ccw': string
-		'rotate-cw': string
-		ruler: string
-		search: string
-		'send-backward': string
-		'send-to-back': string
-		'settings-horizontal': string
-		'settings-vertical-1': string
-		'settings-vertical': string
-		'share-1': string
-		'share-2': string
-		'size-extra-large': string
-		'size-large': string
-		'size-medium': string
-		'size-small': string
-		'spline-cubic': string
-		'spline-line': string
-		'stack-horizontal': string
-		'stack-vertical': string
-		'stretch-horizontal': string
-		'stretch-vertical': string
-		'text-align-center': string
-		'text-align-justify': string
-		'text-align-left': string
-		'text-align-right': string
-		'tool-arrow': string
-		'tool-embed': string
-		'tool-eraser': string
-		'tool-frame': string
-		'tool-hand': string
-		'tool-highlight': string
-		'tool-laser': string
-		'tool-line': string
-		'tool-media': string
-		'tool-note': string
-		'tool-pencil': string
-		'tool-pointer': string
-		'tool-text': string
-		trash: string
-		'triangle-down': string
-		'triangle-up': string
-		twitter: string
-		undo: string
-		ungroup: string
-		'unlock-small': string
-		unlock: string
-		'vertical-align-center': string
-		'vertical-align-end': string
-		'vertical-align-start': string
-		visible: string
-		'warning-triangle': string
-		'zoom-in': string
-		'zoom-out': string
-	}
-	translations: {
-		ar: string
-		ca: string
-		da: string
-		de: string
-		en: string
-		es: string
-		fa: string
-		fi: string
-		fr: string
-		gl: string
-		he: string
-		'hi-in': string
-		hu: string
-		it: string
-		ja: string
-		'ko-kr': string
-		ku: string
-		languages: string
-		main: string
-		my: string
-		ne: string
-		no: string
-		pl: string
-		'pt-br': string
-		'pt-pt': string
-		ro: string
-		ru: string
-		sv: string
-		te: string
-		th: string
-		tr: string
-		uk: string
-		vi: string
-		'zh-cn': string
-		'zh-tw': string
-	}
-	embedIcons: {
-		codepen: string
-		codesandbox: string
-		excalidraw: string
-		felt: string
-		figma: string
-		github_gist: string
-		google_calendar: string
-		google_maps: string
-		google_slides: string
-		observable: string
-		replit: string
-		scratch: string
-		spotify: string
-		tldraw: string
-		vimeo: string
-		youtube: string
-	}
-}
+import { AssetUrlOptions, AssetUrls } from './types'
```

Comment:
Types are now reused, so these `.d.ts` are no longer generated. We only generate `types.d.ts`.